### PR TITLE
feat: add notifyRewardAmount tests for when amount=0

### PR DESCRIPTION
**Pull request by ramenforbreakfast**

### CODE REVIEW

File: apps/smart-contracts/external/yield/test/SingleTokenStakingRewards.js [hunk 1]
```diff
@@ -832,6 +849,37 @@ contract('SingleTokenStakingRewards', (accounts) => {
...
+
+        it('resets period with leftover rewards if reward = 0', async () => {
+          assert.bnEqual(await StakingRewardsDeployed.totalSupply(), toBN(0))
+          const leftoverRewards = await getLeftoverRewards(
+            timeToNotifyRewards,
+            StakingRewardsDeployed
+          )
+          assert.bnGt(leftoverRewards, toBN(0))
+          const newRewardValue = toBN(0)
+          const expectedRewardForDuration = await getExpectedRewardForDuration(
```
Comment (XavierEkkel) [inline apps/smart-contracts/external/yield/test/SingleTokenStakingRewards.js:861]: add assertion that this is also >0
same for the duplicate below

File: apps/smart-contracts/external/yield/test/SingleTokenStakingRewards.js [hunk 2]
```diff
@@ -950,6 +998,23 @@ contract('SingleTokenStakingRewards', (accounts) => {
             lastTimestamp.add(rewardsDuration)
           )
         })
+
+        it('starts new reward of 0 if reward = 0', async () => {
+          assert.bnGt(await StakingRewardsDeployed.totalSupply(), toBN(0))
```
Comment (XavierEkkel) [inline apps/smart-contracts/external/yield/test/SingleTokenStakingRewards.js:1003]: this is in the 'if existing stakers' describe block - shouldn't total supply be > 0?
same for the one below

Comment (ramenforbreakfast) [reply apps/smart-contracts/external/yield/test/SingleTokenStakingRewards.js:1003]: But it is? `bnGt`?

File: apps/smart-contracts/external/yield/test/SingleTokenStakingRewards.js (hunk 1 above)
Comment (ramenforbreakfast) [reply apps/smart-contracts/external/yield/test/SingleTokenStakingRewards.js:861]: Will add this.

File: apps/smart-contracts/external/yield/test/SingleTokenStakingRewards.js (hunk 2 above)
Comment (XavierEkkel) [reply apps/smart-contracts/external/yield/test/SingleTokenStakingRewards.js:1004]: oops I read it as `eq`, ignore this!

**Review by XavierEkkel — APPROVED** [review]
LGTM

**Pull request merged**